### DISCUSSION TITLE
Is there any ways to set baseUrl with routify v3?

### DISCUSSION THREAD

**Discussion by DevYuns** [roxiness/routify#423] [Q&A] [answered]
Hi all!

I'm making a basic static site with `ph-pages`, `routify v3` and `vite`.
My problem is that the project works fine at a first look at production but will throw a 404 error when I tried to reload.

I think this is because `gh-pages url` look like `<organization>.github.io/<repo_name>`. So I set base url to `vite`, but how about `routify`?

I tried to code below, but It isn't work as properly.  the result url looks like `https://svelte-seoul.github.io/svelte-admin-ui/svelte-admin-uiundefined`

p.s `svelte-admin-ui` is my repo name.

```
  import {Router} from '@roxi/routify';
  import routes from '../.routify/routes.default.js';

  const production = process.env.NODE_ENV === 'production';
  const urlRewrite = {
    toExternal: (url: string) => {
      return production ? `/svelte-admin-ui${url}` : url;
    },
    toInternal: (url: string) => {
      return production ? url.replace(/^\/svelte-admin-ui/, '') : url;
    },
  };

  <Router routes={routes} urlRewrite={urlRewrite} />
```

Can anyone help? 😅 If someone wants to see the whole code, refer [here](https://github.com/svelte-seoul/svelte-admin-ui).

**Comment by jakobrosenberg** [comment, marked as answer]
Your index.html isn't configured correct from what I can tell.

view-source:https://svelte-seoul.github.io/svelte-admin-ui/

It's trying to resolve assets from the root `/`, but they should be resolved from `/svelte-admin-ui`. Vite handles the index.html, so you'll have to refer to their documentation.
